mrp_subcontracting_partner_management/models/res_partner.py

```diff
@@ -0,0 +1,267 @@
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    is_subcontractor_partner = fields.Boolean(string="Subcontractor")
+    subcontracted_created_location_id = fields.Many2one("stock.location")
+    partner_picking_type_id = fields.Many2one("stock.picking.type")
```

Comment:
I don't think a new picking type per subcontractor is needed. Too much noise IMO.

Reply:
I mean to use the same picking for all